jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -76,6 +81,10 @@ def __init__(self, *args, **kwargs):
         self.flux_unit = UnitSelectPluginComponent(self,
                                                    items='flux_unit_items',
                                                    selected='flux_unit_selected')
+        self.flux_or_sb = SelectPluginComponent(self,
+                                                items='flux_or_sb_items',
+                                                selected='flux_or_sb_selected',
+                                                manual_options=['Surface Brightness', 'Flux'])
 
     @property
     def user_api(self):
```

Comment:
we'll probably want to expose `flux_or_sb` through the user API here.  And if we decide to rename `flux_unit`, then we'll need to account for backwards compatibility (there are other cases where we do that - see `width`/`continuum_width` in `line_analysis`, for example).

Reply:
My following commit, the `flux_unit` and `sb_unit `will be included with this same methodology as `line_analysis`. Thank you for pointing me in the right direction! I am trying to finalize the case where the units have been translated, and after that, the units are converted (which differentiates from the data collection's data item).